commit f23bf005a879f4fb1e898aef03108138dbbadffc (refs/changes/77/2077/1)
Author: Isaac Davis <isaac.davis@joyent.com>
Date:   2017-06-12T21:32:18+00:00 (2 years, 4 months ago)
    
    OS-6182 dladm segfaults when creating overlay without specifying search plugin

diff --git a/usr/src/cmd/dladm/dladm.c b/usr/src/cmd/dladm/dladm.c
index 40c28ad6ab..797de86319 100644
--- a/usr/src/cmd/dladm/dladm.c
+++ b/usr/src/cmd/dladm/dladm.c
@@ -9925,7 +9925,7 @@ do_create_overlay(int argc, char *argv[], const char *use)
 	if (encap == NULL)
 		die("missing required encapsulation plugin");
 
-	if (encap == NULL)
+	if (search == NULL)
 		die("missing required search plugin");
 
 	if (optind != (argc - 1))
